x/nameservice/keeper/keeper.go

```diff
@@ -45,6 +46,8 @@ var (
 	// PrefixBondIDToAuthoritiesIndex is the prefix for the Bond ID -> [Authority] index.
 	PrefixBondIDToAuthoritiesIndex = []byte{0x06}
 
+	PrefixAttributesIndex = []byte{0x07}
```

Comment:
Let's add a small comment above like the other prefixes e.g. "PrefixAttributesIndex is the prefix for the nameservice Record.Attributes -> Record.ID index".